tests/core/test_algorithm.py

```diff
@@ -10,15 +10,22 @@
     HAVE_SCIPY = True
 
 
+@pytest.fixture
+def rng():
+    import numpy as np
+
+    return np.random.default_rng(seed=42)
+
+
```

Comment:
Move this to conftest.py so it only needs to be defined once?

Reply:
Sounds better, thanks!